source/scenario/scenario_dummy.vwf.yaml

```diff
@@ -166,6 +166,15 @@ children:
           - setObjective:
             - "Explore the Cartesian Plane!"
 
+        onTriangle:
+          triggerCondition:
+            - onBlocklyPolygon:
+              - rover2
+              - [ [21,12] , [22,13] , [22,12] , [21,12] ]
```

Comment:
So, a couple things...

First, for now at least let's assume that all polygons are closed.

Second, let's assume that we don't care either about which vertex they start on or about which direction they trace the vertices in - that is, they can start anywhere and go around the polygon either clockwise or counterclockwise.  If, somewhere down the road, we want to be more strict then we can add boolean parameters to restrict what we accept, but let's handle the most general case right now.